### PR TITLE
Config documentation

### PR DESCRIPTION
**Pull request by ciuckc**
As the title says

### CODE REVIEW

File: docs/Config.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
+# Webserv configuration file documentation
+
+Description and usage of the parameters in the configuration file.
+
+## server
+Creates a new server block. Each server block represents a virtual server. A configuration file can contain multiple server blocks.
+
+### Syntax
+```server{...}```
```
Comment (Lucwousin) [inline docs/Config.md:9]: Change this

**Comment by mbatstra** [comment]
mmm idk if i trust all this

File: docs/Config.md (hunk 1 above)
Comment (ciuckc) [reply docs/Config.md:9]: resolved

**Review by mbatstra — CHANGES_REQUESTED** [review]
sure buddy

**Review by mbatstra — APPROVED** [review]
si

**Pull request merged**